### PR TITLE
TCOTA-576 fix submission results showing for wrong assessment

### PR DESCRIPTION
**Pull request by relong76**

### CODE REVIEW

File: src/main/java/edu/iu/terracotta/service/app/impl/AssessmentServiceImpl.java [hunk 1]
```diff
@@ -132,11 +132,12 @@ private AssessmentDto toDto(Assessment assessment, Participant participant, bool
 
         List<Submission> participantSubmissions = submissionService.findByParticipantId(participant.getParticipantId());
```
Comment (machristie) [inline src/main/java/edu/iu/terracotta/service/app/impl/AssessmentServiceImpl.java:133]: Suggestion: use submissionService.findByParticipantIdAndAssessmentId to do the assessment filtering here

**Review by machristie — CHANGES_REQUESTED** [review]
This is a good catch, @relong76 . Looks good for the most part. I just have one small suggestion.

File: src/main/java/edu/iu/terracotta/service/app/impl/AssessmentServiceImpl.java (hunk 1 above)
Comment (relong76) [reply src/main/java/edu/iu/terracotta/service/app/impl/AssessmentServiceImpl.java]: Thanks! Fixed.

**Review by machristie — APPROVED** [review]
LGTM 👍

**Pull request merged**